tests/utils.js

```diff
@@ -0,0 +1,60 @@
+function mapPositionsToArgs(replacement, positions, staticArg) {
```

Comment:
can we use arrow functions here (in this file) too?

Reply:
also, is it assumed that we have test coverage via the added test files for the rules?

Reply:
Essentially yes, it is somewhat implicit.

I could add test coverage but it would just be validating that the transformation of inputs to outputs is waht the test runner already expects. Only way I could really be sure of that is by then passing the outputs into the test runner, which we already do by just using the function.

Only benefit I could see of covering this code with tests is if we see a known bug which from looking at the outputs, it is behaving as I expect it to.

Wdyty? Should I try to add tests here? Do you know what you would want to test?